spec/test_app_templates/Gemfile.extra

```diff
@@ -1,6 +1,6 @@
 # Use this file to reference specific commits of gems.
 
-gem 'curation_concerns', github: 'projecthydra-labs/curation_concerns', branch: 'master'
+gem 'curation_concerns', github: 'projecthydra-labs/curation_concerns', branch: 'toggle_popover'
```

Comment:
This is the bit I need to revert once https://github.com/projecthydra-labs/curation_concerns/pull/550 is merged, and then I'll force-push what should be the last change required for the `lib/` tidy-up.